docs/sources/httppoller.md

```diff
@@ -0,0 +1,47 @@
+# Event Source for HTTP Polling
+
+This event source launches periodic HTTP requests against an external system endpoint, turning received requests into [CloudEvents][ce] to be consumed by other TriggerMesh components.
+
+## Prerequisites
+
+- An external system that exposes an HTTP endpoint.
+- When using HTTP basic authentication, a secret containing the password.
+
+## Deploying an Instance of the HTTP Poller Source
+
+Open the Bridge creation screen and add a source of type `HTTP Source`.
```

Comment:
Isn't this supposed to be HTTP Poller?

Reply:
Thanks, now you know where I copied it from.
https://github.com/triggermesh/docs/pull/112/commits/eb390089bc41c64b3f1ea8e98ecde6cd19b4ac30

I took the license to also fix a doc page that I submitted very recently and that needs that same reference fixed.
https://github.com/triggermesh/docs/pull/112/commits/10d851f63d73b20c48302692fb6e5953d9ae3655